bitinformation_pipeline/bitinformation_pipeline.py

```diff
@@ -49,10 +65,21 @@ def get_bitinformation(ds, dim=None, mask=None, label=None, overwrite=False):
     if overwrite:
         info_per_bit = {}
         for var in ds.data_vars:
-            # nbits = ds[var].dtype.itemsize * 8
             X = ds[var].values
             Main.X = X
-            info_per_bit[var] = jl.eval("get_bitinformation(X)")
+            if "mask" in kwargs:
+                raise ValueError(
+                    "bitinformation_pipeline does not wrap the mask argument. Mask your xr.Dataset with NaNs instead."
+                )
+            if "dim" in kwargs:
+                if isinstance(kwargs["dim"], str):
+                    kwargs["dim"] = ds[var].get_axis_num(kwargs["dim"]) + 1
```

Comment:
So here we could add

```suggestion
                if isinstance(kwargs["dim"], str):
                      kwargs["dim"] = ds[var].get_axis_num(kwargs["dim"]) + 1
                elif isinstance(kwargs["dim"], list):
                      kwargs["dim"] = [ds[var].get_axis_num(dim) + 1 for dim in kwargs["dim"]]
```

Reply:
but does `bitinformation.jl` understand `dim` as `list`? I would implement a loop over dim not here, rather somewhere else